examples/chip-tool/templates/commands.zapt

```diff
@@ -209,7 +217,11 @@ public:
     Write{{asCamelCased parent.name false}}{{asCamelCased name false}}(): ModelCommand("write")
     {
         AddArgument("attr-name", "{{asDelimitedCommand (asCamelCased name)}}");
+        {{#if (isString type)}}
```

Comment:
Do we need to differentiate here between byte string and not?  Or have we not diverged the storage for those yet?

Reply:
I have updated the PR to handle `OCTET_STRING` and `CHAR_STRING` more gracefully in general.
I end up using `chip::ByteSpan` for both `OCTET_STRING` and `CHAR_STRING` in the API. High level callers/consumers can implement more pleasant types.

For example the objc API is exposing `OCTET_STRING` as `NSData *`, while `CHAR_STRING` are exposed as `NSString *`.

For this case and for `chip-tool` the input type is `char *`.